### PR TITLE
Electrokinetics tutorial

### PR DESCRIPTION
**Pull request by reinaual**
Write diffusion-advection-reaction tutorial.

### CODE REVIEW

File: doc/tutorials/electrokinetics/electrokinetics.ipynb [hunk 1]
```diff
@@ -4,385 +4,708 @@
...
+   "outputs": [],
+   "source": [
+    "species = espressomd.electrokinetics.EKSpecies(\n",
+    "    lattice=lattice, density=0.0, kT=1.0,\n",
+    "    diffusion=DIFFUSION_COEFFICIENT, valency=0.0,\n",
+    "    advection=True, friction_coupling=True,\n",
+    "    ext_efield=[0., 0., 0.], tau=TAU)\n",
+    "system.ekcontainer.add(species)\n",
+    "\n",
+    "species[BOX_L[0] // 2, BOX_L[1] // 2, 0].density = 1.0"
```
Comment (christophlohrmann) [inline doc/tutorials/electrokinetics/electrokinetics.ipynb]: I would put 1/agrid**3 here to make sure there is one unit of material in the system.
Alternatively would using a gaussian as the starting condition reduce numerical errors connected to discretised gradients?

File: doc/tutorials/electrokinetics/electrokinetics.ipynb [hunk 2]
```diff
@@ -4,385 +4,708 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "total_frames = 200\n",
```
Comment (christophlohrmann) [inline doc/tutorials/electrokinetics/electrokinetics.ipynb]: the video takes way too long to be produced. While the simulation is running, students can't use their interactive session to do subsequent tasks. Could the order be switched so this is last? It also seems like the most advanced system.

File: doc/tutorials/electrokinetics/electrokinetics.ipynb [hunk 3]
```diff
@@ -4,385 +4,708 @@
...
+    "\n",
+    "    ax1.imshow(e1)\n",
+    "    ax2.imshow(e2)\n",
+    "    ax3.imshow(p)\n",
+    "    quiver.set_UVC(flowfield[..., 1], flowfield[..., 0])\n",
+    "    \n",
+    "    progress_bar.update()\n",
+    "    \n",
+    "\n",
+    "animation.FuncAnimation(fig, draw_frame, frames=range(total_frames), interval=300)"
```
Comment (christophlohrmann) [inline doc/tutorials/electrokinetics/electrokinetics.ipynb]: would be cool if the boundaries were visible in the video

**Review by christophlohrmann — CHANGES_REQUESTED** [review]
The physics in the tutorial semms good, especially the reaction in turbulent flow looks fantastic. Obviously, the exercises are still missing.
The language/style of the descriptive text needs to be improved, but that's something I could do

**Review by christophlohrmann — COMMENTED** [review]
Exercises are a bit short (especially in the summer school context where we need people to do stuff for 90mins) but cover the essentials.
Text still needs some polishing. I could take it from here

**Comment by christophlohrmann** [comment]
@jngrad @reinaual ready for review

**Review by christophlohrmann — APPROVED** [review]
can I approve my own changes?

**Review by jngrad — APPROVED** [review]

**Pull request merged**